js/global.js

```diff
@@ -844,14 +862,19 @@ jQuery(document).ready(function($) {
 
         $('.ToggleFlyout').removeClass('Open').closest('.Item').removeClass('Open');
         $('.Flyout').hide();
+        $(this).closest('.ToggleFlyout').setFlyoutAttributes();
     });
 
     $(document).delegate(document, 'click', function() {
         if (lastOpen) {
             $('.Flyout', lastOpen).hide();
             $(lastOpen).removeClass('Open').closest('.Item').removeClass('Open');
         }
-        $('.ButtonGroup').removeClass('Open');
+
+        $('.ButtonGroup').each(function(){
```

Comment:
Shouldn't both these calls work on all matched elements and thus not need the `each()`?